packages/shared/routes/dashboard/wallet/views/Security.svelte

```diff
@@ -92,22 +67,10 @@
         })
     }
 
-    function getLedgerDeviceStatus() {
+    function syncLedgerDeviceStatus() {
         isCheckingLedger = true
-        api.getLedgerDeviceStatus($activeProfile?.profileType === ProfileType.LedgerSimulator, {
-            onSuccess(response) {
-                ledgerDeviceStatus = response.payload.type
-                setTimeout(function () {
-                    isCheckingLedger = false
-                }, 500)
-            },
-            onError(e) {
-                ledgerDeviceStatus = LedgerStatus.Disconnected
-                setTimeout(function () {
-                    isCheckingLedger = false
-                }, 500)
-            },
-        })
+        const _onComplete = () => (isCheckingLedger = false)
```

Comment:
This is just a tinytiny design thing, perhaps we could delay setting this to `false` to allow the refresh icon to spin? 
```javascript
const _onComplete = () => {
    clearTimeout(spinnerTimeout)
    spinnerTimeout = setTimeout(() => (isCheckingLedger = false), 500)
}
```